### DISCUSSION TITLE
Exit Group menu entry now permenant - what to do?

### DISCUSSION THREAD

**Discussion by Draise14** [Bforartists/Bforartists#3493] [Ideas]
![image](https://user-images.githubusercontent.com/35357409/231610079-b62a6dba-adcf-47b5-bce9-d0731246ee3c.png)

Blender now shows the Exit Group menu entry, even if it's not a group - it's now permenantly in the header and when you can't go "up" it will grey out. 

Question:
Should we.. revert this? Make this hide when on surface level instead of disable? 

**Commit in Question**
https://projects.blender.org/blender/blender/pulls/106643

**Comment by ReinerBforartists** [comment]
I like the change. But we hide instead of greying out ...

**Comment by Draise14** [comment]
Yeah, might be inconsistent, but might be also be a non issue. Let's keep as is.

**Comment by ReinerBforartists** [comment]
As told, the greyed out could become hidden. Would be more consistent

**Reply by Draise14** [reply]
Ei, completely hidden? Ah, so this should be a task then - which is reverting what Blender did (show the arrow but greyed out)

**Comment by ReinerBforartists** [comment]
No, not remove. But at top level it makes no sense to show the greyed out button. Just show it when you are in a group.

I'll make it a task.
